internal/auth/internal/config/config.go

```diff
@@ -3,14 +3,58 @@ package config
 import (
 	"flag"
 	"os"
+	"time"
 )
 
 type Config struct {
 	RunAddress      string
 	DatabaseAddress string
 	LoggerLevel     string
+	MigrationsPath  string
+	PrivateKey      string
+	PublicKey       string
+	TokenExp        time.Duration
 }
 
+var defaultPrivateKey = `-----BEGIN PRIVATE KEY-----
```

Comment:
It would be better if you don't save this in code

Reply:
It's default value. We cannot work without it, if user doesn't upload necessary key.
And i don't wont save it in file, in the target picture it is unacceptable. :)